kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowViewStub.kt

```diff
@@ -0,0 +1,62 @@
+package com.squareup.workflow.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.view.ViewGroup
+
+/**
+ * A placeholder [View] that can replace itself with ones driven by workflow renderings,
+ * similar to [android.view.ViewStub].
```

Comment:
```suggestion
 * similar to [android.view.ViewStub].
 *
 * ## Usage
 *
 * In the XML layout for your container view, place a [WorkflowViewStub] where you want child renderings
 * to be displayed. E.g.:
 *
 * ```
 * <LinearLayout
 *     android:…
 *     >
 *         < com.squareup.workflow.ui.WorkflowViewStub
 *             android:id="@+id/child_stub"
 *             />
 *    …
 * ```
 *
 * Then in your [LayoutRunner], get the `ViewRegistry` in your constructor:
 *
 * ```
 * class YourLayoutRunner(view: View, private val viewRegistry: ViewRegistry) {
 * ```
 *
 * pull the view out with `findViewById` like any other view:
 *
 * ```
 * private val child = view.findViewById<WorkflowViewStub>(R.id.child_stub)
 * ```
 *
 * and update it in your `showRendering` method:
 *
 * ```
 * override fun showRendering(rendering: YourRendering) {
 *   child.update(rendering.childRendering, viewRegistry)
 * }
 * ```
```

Reply:
Thanks!

Reply:
```kotlin
/**
 * A placeholder [View] that can replace itself with ones driven by workflow renderings,
 * similar to [android.view.ViewStub].
 *
 * ## Usage
 *
 * In the XML layout for your container view, place a [WorkflowViewStub] where
 * you want child renderings to be displayed. E.g.:
 *
 *    <LinearLayout
 *        android:…
 *        >
 *            <com.squareup.workflow.ui.WorkflowViewStub
 *                android:id="@+id/child_stub"
 *                />
 *       …
 *
 * Then in your [LayoutRunner],
 *   - get the `ViewRegistry` in your constructor
 *   - pull the view out with `findViewById` like any other view
 *   - and update it in your `showRendering` method:
 *
 * ```
 *     class YourLayoutRunner(
 *       view: View, private
 *       val viewRegistry: ViewRegistry
 *     ) {
 *       private val child = view.findViewById<WorkflowViewStub>(R.id.child_stub)
 *
 *       override fun showRendering(rendering: YourRendering) {
 *         child.update(rendering.childRendering, viewRegistry)
 *       }
 *     }
 * ```
 */
```